google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/schema/SpannerTableStatements.java

```diff
@@ -158,15 +162,36 @@ private boolean isElementCollection(Table table, Metadata metadata) {
 
     statements.add(createTableString);
 
-    // Hibernate requires the special hibernate_sequence table to be populated with an initial val.
     if (table.getName().equals(SequenceStyleGenerator.DEF_SEQUENCE_NAME)) {
-      statements.add("INSERT INTO " + SequenceStyleGenerator.DEF_SEQUENCE_NAME + " ("
-          + SequenceStyleGenerator.DEF_VALUE_COLUMN + ") VALUES(1)");
+      // Caches the INSERT statement for after the DDL batch.
+      addStatementAfterDdlBatch(
+          metadata,
+          "INSERT INTO " + SequenceStyleGenerator.DEF_SEQUENCE_NAME + " ("
```

Comment:
Not actionable for this PR: even though table creation/update/deletion are not frequent or high-volume operations, the amount of string concatenation (which is slow and creates intermediate literals) is starting to be concerning.
Is that typical for Hibernate and its dialects?

Reply:
Good point, looks like Hibernate uses `StringBuilder` in their code instead of string concatenation. We can definitely switch to this style in the future.

I created an issue to track this: #182